dcicutils/ff_utils.py

```diff
@@ -1488,7 +1488,10 @@ def get_response_json(res):
     it is not present. Used with the metadata functions.
     """
     try:
-        res_json = res.json()
+        if isinstance(res, VirtualAppResponse):
+            res_json = res.json
+        else:
```

Comment:
Again this is most conservative but I wonder if this should be type-checking, too. I guess let's go with this for now as most minimal change.

Reply:
You mean type checking the `json` property?

Reply:
Oh, I was just wondering if we should do a plain else or another if with another type. But it previously just did `.json()` with no test, so this will be most compatible. I wouldn't have designed fresh code that way if I knew multiple types were in play. It preferences one default over another when you really don't konw. The presence of the `try:` around it is an apology for just taking a guess and hopes the damage won't be too great if it guesses wrong.